crates/bifrost/src/loglets/local_loglet/provider.rs

```diff
@@ -86,15 +87,13 @@ impl LogletProvider for LocalLogletProvider {
     }
 
     fn start(&self) -> Result<(), ProviderError> {
-        let writer_options = WriterOptions {
-            channel_size: self.opts.writer_queue_len,
-            commit_time_interval: self.opts.writer_commit_time_interval.into(),
-            batch_size_threshold: self.opts.writer_queue_len,
-            flush_wal_on_commit: self.opts.flush_wal_on_commit,
-            disable_wal: self.opts.rocksdb_disable_wal,
-        };
-
-        let log_writer = self.log_store.create_writer(writer_options).start()?;
+        let mut updateable = Configuration::mapped_updateable(|c| &c.bifrost.local);
+        let opts = &updateable.load().rocksdb;
+        let manual_wal_flush = opts.rocksdb_batch_wal_flushes() && !opts.rocksdb_disable_wal();
```

Comment:
What if `rocksdb_disable_wal` changes later? Since we are passing in the `manual_wal_flush` value, it won't react to these changes.